content/blog/2023/2022blog.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "2022 Progress!"
+author: "ghostway"
+published: 2023-01-14
+---
+
+2022 has been a great year for Leela. A lot of new contributors appeared and made significant improvements; overall, Leela has become considerably stronger and more interesting.
+This year has brought huge changes to how the search was conducted, network architecture, available backends and more.
+<!--more-->
+
+I'll dive into some of them.
+
+#### Network Architecture
+In the AlphaZero paper, Google used Resnets to predict the value and policies of the position. In 2022, *Arcturai* after a lot of work
+presented attention-based policy. Attention is a well known block that was already used in Imagenet (for more details about attention, see [Attention Is All You Need](https://arxiv.org/abs/1706.03762)). A blog on that will come soon™!
+That opened the path to replacing the whole network with a transformer-like architecture. With *danielmonoroe*'s work and improvements -- Smolgen being the major
+improvement so far -- BT2 is currently the strongest network in existence (for details on the new architecture and improvements, [read Daniel's readme](https://github.com/Ergodice/lczero-training/blob/attention-net-body/README.md)).
+
+#### Monte Carlo Graph Search
+In 2020, Crazyara's author and two others released a [paper](https://arxiv.org/abs/2012.11045) describing a way of handling transpositions. Transpositions are a big problem in AlphaZero-like search -- it's just wasting visits and computation!
+Until then, only the NN cache worked. As in, when you know already that a certain position has this policy/value, just use them. But after you created that node, there's no way to share the results between the two. So you end up wasting some part of your time.
+The paper describes a way of sharing data between transpositions that don't necessarily share the same exact state. In 2022, some initial planning for incorporating MCGS into Leela was done, and *Etcaqab* took the challenge. Currently, it outperforms regular Leela and is competing in competitions!
```

Comment:
* Currently, it → It already
* is competing in competitions → participates in competitions

It also makes sense to add a sentence why it's not yet included in the release.
Something like `However, there are still a few issues to iron out before this feature appear in the release (for example, currently it can take a few seconds before the engine starts to think, due to the GC code, and this could cause losing on time in competitions)`.